src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,67 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.util.List;
+
+@Entity
+@Table(name = "cinema_hall")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+    @OneToMany(mappedBy = "cinemaHall", cascade = CascadeType.PERSIST)
+    private List<MovieSession> movieSessions;
```

Comment:
There is no movieSession in CinemaHall entity. See the [Model structure](https://github.com/mate-academy/hibernate-create-movie-session-hw?tab=readme-ov-file#model-structure)